CrossUI/CrossUI.Touch/Dialog/Elements/HtmlElement.cs

```diff
@@ -125,10 +125,7 @@ public override void Selected(DialogViewController dvc, UITableView tableView, N
                 {
                     NetworkActivity = false;
                     vc.NavigationItem.RightBarButtonItem = null;
-                    if (_web != null)
-                        _web.LoadHtmlString(
-                            String.Format("<html><center><font size=+5 color='red'>{0}:<br>{1}</font></center></html>",
-                                          "An error occurred:".GetText(), args.Error.LocalizedDescription), null);
+      </PropertyGroup>LocalizedDescription), null);
```

Comment:
You can undo that, I don't think I worked on that.

On Wed, Mar 11, 2015 at 2:51 PM, Stuart Lodge <notifications@github.com>
wrote:

> In CrossUI/CrossUI.Touch/Dialog/Elements/HtmlElement.cs
> <https://github.com/MvvmCross/MvvmCross/pull/918#discussion_r26259590>:
>
> > @@ -125,10 +125,7 @@ public override void Selected(DialogViewController dvc, UITableView tableView, N
> >                  {
> >                      NetworkActivity = false;
> >                      vc.NavigationItem.RightBarButtonItem = null;
> > -                    if (_web != null)
> > -                        _web.LoadHtmlString(
> > -                            String.Format("<html><center><font size=+5 color='red'>{0}:<br>{1}</font></center></html>",
> > -                                          "An error occurred:".GetText(), args.Error.LocalizedDescription), null);
> > +      </PropertyGroup>LocalizedDescription), null);
>
> Can you check this change - it does look wrong...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MvvmCross/MvvmCross/pull/918/files#r26259590>.
>